src/components/Charts/AnimatedInLineChart.tsx

```diff
@@ -0,0 +1,90 @@
+import { Group } from '@visx/group'
+import { LinePath } from '@visx/shape'
+import { easeCubicInOut } from 'd3'
+import React from 'react'
+import { useEffect, useRef, useState } from 'react'
+import { animated, useSpring } from 'react-spring/web'
+import { useTheme } from 'styled-components/macro'
+
+import { LineChartProps } from './LineChart'
+
+const config = {
+  duration: 800,
+  easing: easeCubicInOut,
+}
+
+// code reference: https://airbnb.io/visx/lineradial
+
+function LineChart<T>({
```

Comment:
Name this `AnimatedInlineChart` to match the filename.
Rename the filename `AnimatedInlineChart` (inline is one word).

Reply:
it is an "animated in line-chart" as opposed to an "animated inline chart" haha